configs/configs.go

```diff
@@ -55,3 +61,51 @@ func LoadConfig(path string, obj interface{}) error {
 	}
 	return nil
 }
+
+// GLogFlags contains the values of the flags that github.com/golang/glog
+// registers.
+type GLogFlags struct {
+	LogToStderr     bool
+	AlsoLogToStderr bool
+	LogDir          string
+	LogBacktraceAt  string
+	V               int
+	VModule         string
+}
+
+// SetGLogFlags sets the flags that github.com/golang/glog registers from
+// the config struct if they contain non-default values so that they can
+// be set from a file.
+func SetGLogFlags(gf GLogFlags) error {
+	if gf.LogToStderr {
+		if err := flag.Set("logtostderr", fmt.Sprintf("%t", gf.LogToStderr)); err != nil {
```

Comment:
Suggestion: Just `fmt.Sprint(gf.LogToStderr)` should work.

Ditto for alsologtostderr and v.